{{cookiecutter.repo_name}}/docs/installation.rst

```diff
@@ -45,14 +84,17 @@ Or download the `tarball`_:
 
 .. code-block:: console
 
-    $ curl  -OL https://github.com/anmut-consulting/{{ cookiecutter.repo_name }}/tarball/latest
+    $ curl -OL https://github.com/anmut-consulting/{{ cookiecutter.repo_name }}/tarball/latest
 
-Once you have a copy of the source, you can install it with:
+.. _tarball: https://github.com/anmut-consulting/{{ cookiecutter.repo_name }}/tarball/latest
```

Comment:
Perhaps add a note stating that `/latest` can be replaced by the version they want (if that's the case...not too sure if github actually creates tarballs for every release).

Reply:
yep, it works for any version release.